README.md

```diff
@@ -202,6 +206,31 @@ class Document
 end
 ```
 
+Some type declaration have options like `store_as_string` for `datetime`
+or `serializer` for `serializable`. It's possible to specify them for elements type:
+
+```ruby
+class Document
+  include DynamoId::Document
+
+  field :values, :set, of: { serialized: { serializer: JSON } }
+end
+```
+
+As far as DynamoDB doesn't allow storing empty strings in Set
+when `Dynamoid` saves a document it removes all empty strings in `set` fields.
+
+#### Note on array type
+
+`array` field declaration supports `of` option as well:
```

Comment:
Is there any guidance we can add relative to why or when one should use a `set` and when one should use an `array`, aside from the element type restrictions?  This may not be the right place for that sort of thing.

Reply:
@pboling 

The guidance could be found in DynamoDB's documentation as far as Dynamoid's types are mapped straightforward to DynamoDB's - array on List and set on Set.

Reply:
As far as I can tell, the mapping you refer to is not in the README anywhere.  Might be in some wiki, but it would be helpful to clarify in the README.  Most Rubyists are not familiar with Ruby's Set class:

| Amazon DynamoBD | Dynamoid gem |
| -------------------- | --------------- |
| List                            | Array                |
| Set                            | Set                    |